_docs/style/focus-states.md

```diff
@@ -3,29 +3,36 @@ title: Focus States
 layout: docs
 category: Style
 usage: |
-  Focused state allows the user to see what interactive element has been clicked on or navigated to using the keyboard. Only elements that can be interacted with can receive focus.
+  Focus visually indicates that the user has clicked on or navigated to a user interface (UI) element. When a UI element is in focus, it is ready to receive input from the user. Focus applies to all interactive elements.
 resource: true
 status: complete
-last-modified: 2018-05-10
+last-modified: 2019-01-11
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-Focus is received by interactive components when the user clicks on them.  Focus is also applied to clickable elements when the user presses the tab key to traverse the UI using the keyboard.
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% endcolumn %}
+## When to use
+
+Apply focus to all interactive elements in a UI.
+
+Do not focus UI components that do not provide interaction feedback, including:
```

Comment:
"UI components that do not provide interaction feedback, including:" is a confusing phrase. 

maybe change to 
"non-interactive elements, including:"

Reply:
Done. woot!